docs/getting-started/installation.md

```diff
@@ -1,4 +1,4 @@
-Decompose provides a number of modules, they all published to Maven Central Repository.
+Decompose provides a number of modules, they are all published to Maven Central Repository. [![Maven Central](https://img.shields.io/maven-central/v/com.arkivanov.decompose/decompose?color=blue)](https://search.maven.org/artifact/com.arkivanov.decompose/decompose)
```

Comment:
Should there be a double line break before the badge?

Reply:
I was thinking the badge was just part of the same sentence, but can move it onto another line if you think it makes more sense to break them up 🤷‍♂️

Reply:
In the screenshot, it doesn't fit on one line. I would wrap the line of you don't mind. Thanks!

Reply:
Never mind, I think it's fine.

Reply:
Was just about to break it up into another line today, let me know since its an easy fix.

Reply:
Nothing to worry about, this is not so important 👍